src/index.js

```diff
@@ -9,14 +9,48 @@
  * OF ANY KIND, either express or implied. See the License for the specific language
  * governing permissions and limitations under the License.
  */
+const initfastly = require('@adobe/fastly-native-promises');
 
 /**
- * This is the main function
- * @param {string} name name of the person to greet
- * @returns {string} a greeting
+ * authenticates token and service with Fastly
+ *
+ * @param {string} token Fastly Authentication Token
+ * @param {string} service serviceid for a helix-project
+ */
+async function authFastly(token, service) {
+  // verify Fastly credentials
+  try {
+    const Fastly = await initfastly(token, service);
+    await Fastly.getVersions();
```

Comment:
why is this important?

Reply:
The call to initfastly just initializes the fastly client. For some reason it doesn't really depend on valid credentials until you wanna use a client method

Thanks for looking at this with a careful eye! :)